public/src/reducers/calls.jsx

```diff
@@ -81,6 +83,11 @@ export function volunteerCallsReducer(state = defaultCalls, action) {
         ...state,
         call_volunteer_active: false
       };
+    case SET_NO_CALLS_AVAILABLE:
+      return {
+        ...state,
+        no_calls_available: payload
```

Comment:
pending on previous question: from my previous comment, this could just set 'no_calls_available' to true explicitly... if the toggle to false isn't necessary.